loops-and-arrays.js

```diff
@@ -5,6 +5,9 @@
 // ==========================================
 
 const names = ["Henk", "Piet", "Fred", "Joop"];
+    for (let i = 0; i < names.length; i++) {
+        console.log(names[i] + "je");
+}
```

Comment:
let op: het verwachte antwoord is console.log(names) geeft: ["Henkje", "Pietje", "Fredje", "Joopje"]
hij wil dat je een array uitprint met de aangepaste namen erin. dit betekent dat je in regel 9 niet moet loggen maar moet  overschrijven. names[i] = names[i] + "je". buiten je for loop zeg je dan console.log(names).


dit wordt ongeveer bij elke opdracht gevraagd in loops and arrays. misschien leuk om nog aan te passen?

Reply:
Thanks! Ik ga dit zeker nog aanpassen